src/pages/SettingsPage.js

```diff
@@ -86,6 +113,25 @@ const SettingsPage = ({
                         {Str.removeSMSDomain(session.email)}
                     </Text>
                     )}
+                    <View style={[styles.settingsPageBody, styles.mt4]}>
```

Comment:
I think it would be nice to add some margin bottom to this view so that there is more separation between this form field block and the sign out button. Perhaps just try using `mb5` but we might need to make a larger margin bottom style if the space isn't quite large enough.

Reply:
mb5:
![image](https://user-images.githubusercontent.com/47184118/109482939-86ba0680-7a76-11eb-9d1e-c65ed7eaccd6.png)

mb6:
![image](https://user-images.githubusercontent.com/47184118/109482890-7b66db00-7a76-11eb-9dad-71f785eea59e.png)

Reply:
mb6 already exists in the spacing.js 👍 
let me know which one you want and I'll make the commit

Reply:
Let's do mb6, thanks!